.github/workflows/release.yml

```diff
@@ -183,6 +184,7 @@ jobs:
           >>$GITHUB_ENV echo SPEC=$(<${JSON} jq -r .runtimes.compact.subwasm.core_version.specVersion)
 
       - name: Upload compressed ${{ matrix.runtime.name }} v${{ env.SPEC }} wasm
+        # TODO: Replace as it has been deprecated
         uses: actions/upload-release-asset@v1
         env:
```

Comment:
From the [action's readme](https://github.com/actions/upload-release-asset?tab=readme-ov-file#github-action---releases-api) the new action that seems to replace this one is [softprops/action-gh-release](https://github.com/softprops/action-gh-release)